src/internal/tester/tconfig/config.go

```diff
@@ -45,6 +46,7 @@ const (
 	EnvCorsoM365TestSiteURL          = "CORSO_M365_TEST_SITE_URL"
 	EnvCorsoM365TestTeamID           = "CORSO_M365_TEST_TEAM_ID"
 	EnvCorsoM365TestGroupID          = "CORSO_M365_TEST_GROUP_ID"
+	EnvCorsoM365TestChannelID        = "CORSO_M365_TEST_CHANNEL_ID"
```

Comment:
I don't think it's in our interest to use these envs for any entity more granular than a resource (user, site, etc).  This is basically like using these envs to specify containers.  We already have selectors as a system to handle that granularity, and prior work has shown that looking for expected containers by name (ex: a site drive named "documents") is an effective management strategy.  Adding that identification here could both couple test expectations together unnecessarily, and increase the complexity and maintenance of baseline testing.

Reply:
okay. In that case we can safely remove the test setup for channels. And use the name to find channels if needed in test cases.